templates/single.twig

```diff
@@ -91,6 +91,19 @@
 						{{ fn('do_blocks', post.take_action_boxout )|raw }}
 					{% endif %}
 				</article>
+				{% if ( post.tags ) %}
+					{% for tag in post.tags %}
```

Comment:
These tags do not have a wrapper element, which probably makes alignment harder.
![Screenshot from 2022-02-28 12-52-32](https://user-images.githubusercontent.com/7604138/155978999-df728606-9f29-4efd-bc01-ea9561b6793a.png)